balancer-js/src/modules/pools/pool-types/concerns/linear/priceImpact.concern.ts

```diff
@@ -3,14 +3,16 @@ import { PriceImpactConcern } from '../types';
 import { Pool } from '@/types';
 
 export class LinearPriceImpact implements PriceImpactConcern {
-  bptZeroPriceImpact(pool: Pool, amounts: bigint[]): bigint {
+  bptZeroPriceImpact(pool: Pool, tokenAmounts: bigint[]): bigint {
     throw new Error('Linear Pool bptZeroPriceImpact Not Implented');
   }
   calcPriceImpact(
     pool: Pool,
     tokenAmounts: bigint[],
-    bptAmount: bigint
+    bptAmount: bigint,
+    isJoin: boolean
   ): string {
-    throw new Error('Linear Pool Price Impact Not Implented');
+    // Linear pools don't have price impact
+    return '0';
```

Comment:
@johngrantuk I'm not completely sure this is true, but if I'm not mistaken Linear pools shouldn't ever have priceImpact, right?